### DISCUSSION TITLE
Работа с ObjectManager, LoadObjectManager, RemoteObjectManager

### DISCUSSION THREAD

**Discussion by Ramsly** [yandex-maps-unofficial/vue-yandex-maps#205] [Q&A (2.x)] [answered]
Здравствуйте! Есть ли реализация с ObjectManager, LoadObjectManager, RemoteObjectManager? Не совсем понятно как работать с большим кол-вом точек на карте (~20 000). Могли бы вы объяснить? Спасибо!

**Comment by daniluk4000** [comment]
Здравствуйте! Для библиотеки версии 2.х вам следует использовать кластеризатор: https://yandex-maps-unofficial.github.io/vue-yandex-maps/components/modules/clusterer.html

**Comment by daniluk4000** [comment, marked as answer]
Я поигрался с кластером на ваших количествах, и остался разочарован производительностью.

Подебажил пару часиков, сильно его ускорил. В следующем патче (rc.12) ждите улучшения.

**Reply by Ramsly** [reply]
Увидел в вашем ТГ канале, получилось круто! Спасибо! Но все же странно, то что Яндекс не реализовали ObjectManager, LoadObjectManager, RemoteObjectManager
